lib/insights/src/lib/analytics-process/components/widgets/number/number.widget.html

```diff
@@ -1,5 +1,5 @@
 <div class="adf-number-widget" [formGroup]="formGroup">
-    <mat-form-field class="example-full-width" [floatLabel]="'always'">
+    <mat-form-field class="example-full-width">
```

Comment:
can this class be renamed to something proper? the "example-" prefix seems like some old copy-paste :)